VTUL/iawa
# change misnamed h21 to h2

**change misnamed h21 to h2.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1933) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
A brief description of what the intended result of the PR will be and/or what problem it solves.

# What's the changes? (:star:)
* Change head 2 tag in filename 

# How should this be tested?
* Build vm with LIBTD-1933 branch and view About page. 
* Ensure that h2 tag renders properly

# Additional Notes:
* branch: LIBTD-1933

# Interested parties
Tag (@ mention) interested parties
@andreaWaldren 

(:star:) Required fields
